lib/stats-output.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Debug = require('broccoli-debug');
+
+module.exports = class StatsOutput extends Debug {
+
+  constructor(inputNode, options) {
+    return super(inputNode, {
+      label: options.name,
+      baseDir: options.dir,
+      force: false
+    });
+  }
+
+  build() {
+    this._shouldSync = this.isEnabled();
```

Comment:
Downside here is that this is using an apparently private property to activate the debug output (used to write a copy of the files). But I didn't find a better way, other than fiddling around with `BROCCOLI_DEBUG`...

Reply:
ya, but I think its fine.

Reply:
I agree that it’s fine, but I’d love a test case added to broccoli-debug’s test suite to make sure I don’t break the basic concept accidentally.